### PR TITLE
Make sure that example LaTeX file is built

### PR DESCRIPTION
**Pull request by kathoef**
To prevent potential future minted problems similar to #1.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: docker://exaesmwp4/texlive:latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: cd tex/ && latexmk -pdf -pdflatex='pdflatex --shell-escape' example.tex
```
Comment (willirath) [inline .github/workflows/main.yml]: `example.tex` is not in `tex/`

Comment (kathoef) [reply .github/workflows/main.yml]: This is what happens for mindless copy and paste actions... thanks.

**Comment by kathoef** [comment]
For this to be of any use though, the Docker image should be build by the Github action and not fetched from Dockerhub. The fetched variant is very likely not the version that should actually be tested, as pending times on Dockerhub are rather long.

**Comment by willirath** [comment]
We have a workflow that should cover almost all of this here: <https://github.com/ESM-VFC/esm-vfc-stacks/blob/master/.github/workflows/repo2docker.yml> (swapping the `repo2docker` step for `docker build`).

**Comment by willirath** [comment]
I'd do the following:

- Add steps that build the docker image here on
  - every PR to master
  - on branch master
- Add step that pushes the docker image to Dockerhub if on master (needs access token stored in secrets)

**Comment by willirath** [comment]
@kathoef Before merging, we need to make sure that 
- I can push to dockerhub 
- and that automatic builds are disabled there

**Pull request merged**